schemas/Manual.locations.schema.json

```diff
@@ -26,76 +26,75 @@
                     "description": "The unique name of the location.",
                     "type": "string"
                 },
-                "region": {
-                    "description": "(Optional) The name of the Region this location is part of.",
-                    "type": "string"
-                },
                 "category": {
-                    "description": "(Optional) An array of the Category(ies) this location is a part of.",
+                    "description": "(Optional) A list of categories to be applied to this location.",
                     "type": "array",
                     "items": {
-                        "$ref": "#/definitions/Category"
+                        "type": "string"
                     },
                     "uniqueItems": true
                 },
                 "requires": {
-                    "description": "(Optional) Either an array of items or a boolean logic string(check discord).",
+                    "description": "(Optional) A boolean logic string that describes the required items, counts, etc. needed to reach this location.",
                     "type": [ "string", "array" ],
                     "items": {
                         "$ref": "#/definitions/Require"
                     },
                     "uniqueItems": true
                 },
-                "place_item_category": {
-                    "description": "(Optional) Specify the category of item that will be placed at this location. (Ignore Logic)",
+                "region": {
+                    "description": "(Optional) The name of the region this location is part of.",
+                    "type": "string"
+                },
+                "place_item": {
+                    "description": "(Optional) Places an item that matches one of the item names listed in this setting at this location. Does not check logical access to the location.",
                     "type": "array",
                     "items": {
-                        "$ref": "#/definitions/Place_item_category"
+                        "type": "string"
                     },
                     "minItems": 0,
                     "uniqueItems": true
                 },
-                "place_item": {
-                    "description": "(Optional) Specify the list of item that can be placed at this location (will only randomly choose 1).",
+                "dont_place_item": {
+                    "description": "(Optional) Places an item that DOES NOT match one of the item names listed in this setting at this location. Does not check logical access to the location.",
```

Comment:
`dont_place_item_*` work in 2 ways, 
1. when  combined with `place_item_*` it remove from the placement list and thus ignore logic 
2. when alone, all it does is call forbid and does not force an item out of logic there thus it does check access if I understand how AP work

its not `dont_place_item_*` that force place an item basicaly, its `place_item_*`
All they do is blacklist items from beeing placed there, the way its done just depend if place_item is present or not

Reply:
Oh no, this was just a lazy copy-paste. Wasn't intending to fix a lot of pre-existing poor wording on schema descriptions when reorganizing the files and supplying defaults, but here we are. 😄 

I've clarified both of these to indicate that it configures what item criteria should not end up at the location during normal gen. Let me know if you prefer a different explanation!